recipes/soci/all/conanfile.py

```diff
@@ -24,10 +24,7 @@ class SociConan(ConanFile):
         "fPIC":             [True, False],
         "empty":            [True, False],
         "with_sqlite3":     [True, False],
-        "with_db2":         [True, False],
```

Comment:
@AbrilRBS  @uilianries  - I can't find anywhere in the descriptions or the comments as to why these lines, which are unrelated to the PR goal/description, are here? Maybe I'm missing something

Reply:
Makes sense - this was confusing when reviewing as was not mentioned anywhere